cumulusci/core/tests/test_flowrunner.py

```diff
@@ -86,11 +86,9 @@ def test_each_flow(self):
                 flow_config = self.project_config.get_flow(flow_name)
                 flow = FlowCoordinator(self.project_config, flow_config, name=flow_name)
             except Exception as exc:
```

Comment:
🧐

Reply:
Not sure what this is about... I was just subbing in f-strings 😂